android/jni/com/mapswithme/maps/PrivateVariables.cpp

```diff
@@ -22,10 +22,10 @@ extern "C"
     return env->NewStringUTF(MY_TRACKER_KEY);
   }
 
-  JNIEXPORT jstring JNICALL
+  JNIEXPORT jint JNICALL
```

Comment:
`/Users/a.zacepin/Downloads/android-ndk-r10e/toolchains/llvm-3.6/prebuilt/darwin-x86_64/bin/clang++ -MMD -MP -MF ./obj/local/armeabi-v7a-hard/objs/mapswithme/com/mapswithme/maps/PrivateVariables.o.d -gcc-toolchain /Users/a.zacepin/Downloads/android-ndk-r10e/toolchains/arm-linux-androideabi-4.8/prebuilt/darwin-x86_64 -fpic -ffunction-sections -funwind-tables -fstack-protector-strong -Wno-invalid-command-line-argument -Wno-unused-command-line-argument -no-canonical-prefixes -fno-integrated-as  -target armv7-none-linux-androideabi -march=armv7-a -mfpu=vfpv3-d16 -mhard-float -D_NDK_MATH_NO_SOFTFP=1 -fno-exceptions -fno-rtti -mthumb -Os -g -DNDEBUG -fomit-frame-pointer -fno-strict-aliasing -Ijni/../../ -I/Users/a.zacepin/Downloads/android-ndk-r10e/sources/cxx-stl/llvm-libc++/libcxx/include -I/Users/a.zacepin/Downloads/android-ndk-r10e/sources/cxx-stl/llvm-libc++/../llvm-libc++abi/libcxxabi/include -I/Users/a.zacepin/Downloads/android-ndk-r10e/sources/android/support/include -Ijni -DANDROID -ffunction-sections -fdata-sections -Wno-extern-c-compat -fvisibility=hidden -O3 -Wa,--noexecstack -Wformat -Werror=format-security -std=c++11 -fvisibility-inlines-hidden -frtti -fexceptions  -Ijni/../../3party/boost -Ijni/../../3party/protobuf/src -Ijni/../../3party/glm -DRELEASE -D_RELEASE -DOMIM_PRODUCTION -Wno-deprecated-register  -I/Users/a.zacepin/Downloads/android-ndk-r10e/platforms/android-15/arch-arm/usr/include -c  jni/com/mapswithme/maps/PrivateVariables.cpp -o ./obj/local/armeabi-v7a-hard/objs/mapswithme/com/mapswithme/maps/PrivateVariables.o 

jni/com/mapswithme/maps/PrivateVariables.cpp:28:12: error: cannot initialize return object of type 'jint' (aka 'int') with an lvalue of type 'const char [6]'
    return MY_TARGET_KEY;
           ^~~~~~~~~~~~~
jni/../../private.h:11:25: note: expanded from macro 'MY_TARGET_KEY'
  #define MY_TARGET_KEY "26547"
                        ^~~~~~~
1 error generated.
make: *** [obj/local/armeabi-v7a-hard/objs/mapswithme/com/mapswithme/maps/PrivateVariables.o] Error 1
make: *** Waiting for unfinished jobs....

:ndkBuildRelease FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':ndkBuildRelease'.
> Process 'command '/Users/a.zacepin/Downloads/android-ndk-r10e/ndk-build'' finished with non-zero exit value 2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.`